src/OpenLoco/Audio/Audio.cpp

```diff
@@ -613,7 +613,7 @@ namespace OpenLoco::Audio
                     return;
                 }
 
-                volume += calculateVolumeFromViewport(id, { loc.x, loc.y }, *viewport);
+                volume += calculateVolumeFromViewport(id, { loc.x, loc.y, loc.z }, *viewport);
```

Comment:
We should probably add this change to the changelog at least.

Reply:
Added the entry